acceptance/tests/parser_functions/puppet_lookup_cmd.rb

```diff
@@ -2,6 +2,9 @@
 # doc:
 # https://docs.puppetlabs.com/puppet/latest/reference/lookup_quick_module.html
 
+host = agents.find { |host| host != master }
+skip_test "No agents present that are not running on a master" if host.nil?
+
```

Comment:
I thought the `lookup` command was only applicable when running on the same host as the master, so I'm not sure why it's important for the master to have an agent running on it?